app/components/top_controls.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 
 class Components::TopControls < Components::Base
-  def initialize(book:, files:)
+  def initialize(book:, files:, page: nil)
     @book = book
     @files = files
+    @page = page
   end
 
   def view_template
-    ControlsBar do |bar|
+    ControlsBar(
+      data: {
+        top_controls_hide_tashkeel_text_value: t(".hide_tashkeel"),
+        top_controls_show_tashkeel_text_value: t(".show_tashkeel"),
+        top_controls_hide_tashkeel_tooltip_value: t(".hide_tashkeel_tooltip"),
+        top_controls_show_tashkeel_tooltip_value: t(".show_tashkeel_tooltip")
+      }
```

Comment:
I don't know why, but I putting the controller values in the `Pages#show` action view [here](https://github.com/ieasybooks/aljam3-web-app/blob/main/app/views/pages/show.rb#L27-L30). I liked where you put them. Could you please move the other values from the `Pages#show` action view to this component? Just the `top-controls` controller values. We can handle the other values in another PR.